frontend/src/routes/(authenticated)/org/[org_id]/+page.ts

```diff
@@ -315,14 +316,26 @@ export async function _deleteOrg(orgId: string): $OpResult<DeleteOrgMutation> {
   return result;
 }
 
-export async function _getMyProjects(): Promise<LoadMyProjectsQuery['myProjects']> {
+export async function _getProjectsIManage(user: LexAuthUser): Promise<LoadMyProjectsQuery['myProjects']> {
   const client = getClient();
+
   //language=GraphQL
   const results = await client.query(graphql(`
-        query loadMyProjects {
-            myProjects(orderBy: [
+        query loadMyProjects($userId: UUID!) {
+            myProjects(
+              orderBy: [
                 {name: ASC }
-            ]) {
+              ],
+              where: {
+                users: {
+                  some: {
+                    and: [
```

Comment:
this can be simplified to 
`some: { userId: {eq: $userId}, role: {eq: MANAGER} }` the `and` is not needed in this context